lib/registration.pm

```diff
@@ -135,7 +135,7 @@ sub accept_addons_license {
     # license (see bsc#1089163) and so are not be shown twice
     push @addons_with_license, @SLE15_ADDONS_WITHOUT_LICENSE unless (is_sle('15+') || is_sle_micro);
     # HA and WE have licenses when calling yast2 scc
-    push @addons_with_license, @SLE15_ADDONS_WITH_LICENSE_NOINSTALL if (is_sle('15+') and get_var('IN_PATCH_SLE'));
+    push @addons_with_license, @SLE15_ADDONS_WITH_LICENSE_NOINSTALL if (is_sle('<=15-sp4') and get_var('IN_PATCH_SLE'));
```

Comment:
It seems changed the scope, I don't think this should make SLE12 match the condition. Do you mean >=15SP5 it won't have licenses?
If that then the condition should be '>=15' and '<=15-SP4' , WDYT?

Reply:
yes, I wrongly assumed is_sle(<=15-sp4) would apply only for sle15. I will change it.